tests/fix_psycopg.py

```diff
@@ -24,7 +24,7 @@ def global_adapters():
 
 
 @pytest.fixture
-@pytest.mark.crdb_skip("2-phase commit")
+# @pytest.mark.crdb_skip("2-phase commit")
```

Comment:
see: https://docs.pytest.org/en/stable/deprecations.html#applying-a-mark-to-a-fixture-function

Reply:
This is totally unrelated with this merge request.

Reply:
Also, if it doesn't work, the line should be removed, not commented out. But please make a separate MR for it.

Reply:
I have seen now that this change has broken the tests. I'll push a commit to master to fix them.

Reply:
Master is now fixed, this changeset can be dropped.